tests/gen_keys.sh

```diff
@@ -38,6 +38,7 @@ ssh-keygen -t rsa     -b 2048 -N ''                 -C 'key_rsa_openssh'
 ssh-keygen -t rsa     -b 2048 -N "${pw}"            -C 'key_rsa_aes256gcm'       -f 'key_rsa_aes256gcm'     -Z aes256-gcm@openssh.com
 ssh-keygen -t rsa     -b 4096 -N ''                 -C 'key_rsa_signed'          -f 'key_rsa_signed'
 ssh-keygen                    -I "${id}" -n "${pr}" -s 'openssh_server/ca_rsa'      'key_rsa_signed.pub'
+ssh-keygen -t rsa     -b 4096 -N ''                 -C 'key_rsa_sha2_256_signed' -f 'key_rsa_sha2_256_signed'
```

Comment:
Please also include the command that created `tests/key_rsa_sha2_256_signed-cert.pub`.